src/plugin/file/kml/ui/kmlui.js

```diff
@@ -110,6 +110,9 @@ plugin.file.kml.ui.createOrEditPlace = function(options) {
   };
 
   var label = (options['feature'] ? 'Edit' : 'Add') + ' Place';
+  if (os.ui.window.exists(windowId)) {
```

Comment:
This currently causes strange behavior if I start creating a new place, then click edit on an existing place (the create window is brought to the front, when I'm expecting to get an edit window). Similarly if I click edit on one place, then click edit on another or try creating a new place.

This should use a unique id for creating a new place, and editing each existing place. I would suggest using the existing id for create, and for the edit case append `ol.getUid(options['feature'])` to the id.

Reply:
Sounds good. Random string was added to allow multiple new places to be created at the same time.